client/landing/subscriptions/components/sort-controls/sort-controls.tsx

```diff
@@ -0,0 +1,56 @@
+import { useTranslate } from 'i18n-calypso';
+import React, { useState, useEffect, ChangeEvent, ReactElement } from 'react';
+import './styles.scss';
+
+export type Option = {
+	label: string;
+	value: string;
+};
+
+type SortControlsProps< T > = {
+	options: Option[];
+	onChange?: ( sortOrder: T ) => void;
+	value: T;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => {};
```

Comment:
If the function returns undefined the lint disable can be removed.

Reply:
Oh, cool! I did not think of this one. 🙂